Posts/2021/04/NewTemporaryFoldersPowerShell

```diff
@@ -0,0 +1,89 @@
+---
+post_title: Borrowing a Built in PowerShell to create a Temporary Folder
```

Comment:
A built-in what? ALso, use sentence case so.

Borrowing a built-in PowerShell command to create a temporary folder